packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/details/services/PropertiesEventProcessorFactory.java

```diff
@@ -82,42 +81,45 @@ public PropertiesEventProcessorFactory(IPropertiesDescriptionService propertiesD
     }
 
     @Override
-    public boolean canHandle(IRepresentationConfiguration configuration) {
-        return configuration instanceof PropertiesConfiguration;
+    public boolean canHandle(String representationId) {
+        return representationId.startsWith("details://");
     }
 
     @Override
-    public Optional<IRepresentationEventProcessor> createRepresentationEventProcessor(IRepresentationConfiguration configuration, IEditingContext editingContext) {
-        if (configuration instanceof PropertiesConfiguration propertiesConfiguration) {
-
-            List<PageDescription> pageDescriptions = this.propertiesDescriptionService.getPropertiesDescriptions();
-            var objects = propertiesConfiguration.getObjectIds().stream()
-                    .map(objectId -> this.objectService.getObject(editingContext, objectId))
-                    .flatMap(Optional::stream)
-                    .toList();
-            if (!objects.isEmpty()) {
-                Optional<FormDescription> optionalFormDescription = Optional.empty();
-                if (!pageDescriptions.isEmpty()) {
-                    optionalFormDescription = new DetailsViewFormDescriptionAggregator().aggregate(pageDescriptions, objects, this.objectService);
-                }
-                FormDescription formDescription = optionalFormDescription.orElse(this.propertiesDefaultDescriptionProvider.getFormDescription());
-
-                FormCreationParameters formCreationParameters = FormCreationParameters.newFormCreationParameters(propertiesConfiguration.getId())
-                        .editingContext(editingContext)
-                        .formDescription(formDescription)
-                        .object(objects.get(0))
-                        .selection(objects)
-                        .build();
-
-                IRepresentationEventProcessor formEventProcessor = new FormEventProcessor(
-                        new FormEventProcessorConfiguration(editingContext, this.objectService, formCreationParameters, this.widgetDescriptors, this.formEventHandlers),
-                        this.subscriptionManagerFactory.create(),
-                        this.representationSearchService,
-                        this.representationRefreshPolicyRegistry,
-                        this.formPostProcessor);
-
-                return Optional.of(formEventProcessor);
+    public Optional<IRepresentationEventProcessor> createRepresentationEventProcessor(String representationId, IEditingContext editingContext) {
+        List<PageDescription> pageDescriptions = this.propertiesDescriptionService.getPropertiesDescriptions();
+
+        var  objectsIdsString = representationId.substring(representationId.indexOf("objectIds=[") + 11);
```

Comment:
Check that the index are not -1 and the string is long enough for a substring, right now this seems very brittle